### PR TITLE
Update add supplier command in user guide

### LINKED ISSUE #227: Update add supplier parameters in user guide

### PR DESCRIPTION
**Pull request by ngjianyi**
Closes #227

### CODE REVIEW

**Review by LinkesV — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -209,13 +227,13 @@ A success message will be displayed if the supplier is successfully marked as ac
...
+Format: `find -s n/NAME com/COMPANY pro/PRODUCT`
 
 Parameters:
 
-- `n/KEYWORD_SUPPLIER_NAME`: This will check if the Supplier's name contains the given keyword
-- `on/KEYWORD_SUPPLIER_COMPANYE`: This will check if the Supplier's company contains the given keyword
-- `pro/KEYWORD_SUPPLIER_PRODUCT`: This will check if the Supplier's product(s) contains in the given keyword
+- `n/NAME`: Must be alphanumeric, and must not be blank.
+- `on/COMPANY`: Must be alphanumeric, and must not be blank.
+- `pro/PRODUCT`: Must be alphanumeric, can include spaces but must not start with a space, and must be between 1 and 50 (inclusive) characters long.
```
Comment (CJianzhi) [inline docs/UserGuide.md:236]: I think the product can start with a space due to the trimming being done

File: docs/UserGuide.md [hunk 2]
```diff
@@ -371,7 +389,7 @@ Parameters:
 - `on/DELIVERY_DATE_TIME`: Must be in dd-mm-yyyy hh:mm format and must not be blank.
 - `stat/STATUS`: Must be one of the following: PENDING, DELIVERED, CANCELLED and must not be blank.
 - `s/SUPPLIER_INDEX`: Must be a number greater than 0 and must not be blank.
-- `pro/PRODUCT`: Must only consist of alphanumeric characters and must not be blank.
+- `pro/PRODUCT`: Must be alphanumeric, can include spaces but must not start with a space, and must be between 1 and 50 (inclusive) characters long.
```
Comment (CJianzhi) [inline docs/UserGuide.md:392]: Same for here

File: docs/UserGuide.md [hunk 3]
```diff
@@ -131,40 +136,53 @@ Adds a supplier to the address book.
...
 <box type="tip" seamless>
 
+Parameters:
+
+- `n/NAME`: Must be alphanumeric, and must not be blank.
+- `p/PHONE`: Must be numeric, and must contain at least 3 digits.
+- `e/EMAIL`: Must be a valid email address, and must not be blank.
+- `com/COMPANY`: Must be alphanumeric, and must not be blank.
+- `[t/TAG]`: Must be alphanumeric, and must be between 1 and 50 (inclusive) characters long.
+- `[pro/PRODUCT]`: Must be alphanumeric, can include spaces but must not start with a space, and must be between 1 and 50 (inclusive) characters long.
```
Comment (CJianzhi) [inline docs/UserGuide.md:146]: Product can start with space

**Review by CJianzhi — COMMENTED** [review]
LGTM! Just some minor adjustments

File: docs/UserGuide.md [hunk 4]
```diff
@@ -209,13 +227,13 @@ A success message will be displayed if the supplier is successfully marked as ac
...
-Format: `find -s n/KEYWORD_SUPPLIER_NAME com/KEYWORD_SUPPLIER_COMPANY pro/<KEYWORD_SUPPLIER_PRODUCT>`
+Format: `find -s n/NAME com/COMPANY pro/PRODUCT`
 
 Parameters:
 
-- `n/KEYWORD_SUPPLIER_NAME`: This will check if the Supplier's name contains the given keyword
-- `on/KEYWORD_SUPPLIER_COMPANYE`: This will check if the Supplier's company contains the given keyword
-- `pro/KEYWORD_SUPPLIER_PRODUCT`: This will check if the Supplier's product(s) contains in the given keyword
+- `n/NAME`: Must be alphanumeric, and must not be blank.
+- `on/COMPANY`: Must be alphanumeric, and must not be blank.
```
Comment (CJianzhi) [inline docs/UserGuide.md:235]: is on correct?

**Pull request merged**